### ISSUE TITLE
Form export page UI

### ISSUE THREAD

**Issue by robertsteilberg** [mVax-app/mVax#16]
Users during testing expressed concern over lack of instructions on the export forms page. The suggestion was made to provide a Help tool-tip with some basic instructions. This would essentially be putting the user documentation inside the app which would be helpful for this less than perfectly intuitive functionality.

It is not intuitive that the "Export Forms" button is on the dashboard page. "Dashboard" is a name that really says nothing and that is part of the problem.

Acceptance criteria: Help button on the Export Forms page which refers to the user docs for the exporting forms process.

Figure out a way to ensure that it becomes intuitive the button would be there. User Testing will need to be undertaken after this is accepted and depending on the outcome of that, this process may have to repeat.

**Comment by tribby5** [comment]
The renaming of the "Dashboard" tab is listed under a separate issue: https://github.com/mVax-app/mVax/issues/17
So will no longer be part of this issue. This issue is going to be closed because the help button is on the export forms page: https://github.com/mVax-app/mVax/pull/55
